### ISSUE TITLE
Method menu defaulting to 'By Hand' in 'Loan Invoice Query1' in Specify 7 (a11y) version

### ISSUE THREAD

**Issue by realVinayak** [specify/specify7#1824]
**Describe the bug**
The Method menu is defaulting to 'By Hand' in 'Loan Invoice Query1' in Specify 7 (a11y) version. However, the Method for the same query in the edge version is set to 'Any'. The query link is: `specify/query/20`. The database is `Ornithology` and the username is `spbirdadmin`.

**To Reproduce**
1. Log in to a11y version of Specify 7 using username `spbirdadmin`. Select `Ornithology `database and go to 'Loan Invoice Query1.
2. One of the queries is `Shipments -> Method -> Equal -> By Hand`.
3. Repeat the above steps in edge version, and the same query is `Shipments -> Method (Any)`

**Expected behavior**
In the a11y version, the Method should be set to 'Any' instead of 'Equal to By Hand'.

**Screenshots**
The left part is edge version and the right part is a11y version for the same query

![shipment_method_loan_invoice_query](https://user-images.githubusercontent.com/61122018/178767256-c57b4496-ea8d-4e7c-b748-b9751c3e4e53.PNG)

**Comment by maxxxxxdlp** [comment]
Good find!

This is the same issue as #1823 so please add a comment there rather than open a new issue
